### DISCUSSION TITLE
如何同时打开两个相同的库？？ 谢谢🙏

### DISCUSSION THREAD

**Discussion by howardmi5** [obsidianzh/forum#62] [Q&A常见问题]
需要在两块屏幕中打开相同的库。
两块屏幕设置是 屏幕延伸（由于一定原因 没有设置为 屏幕2合一）
屏幕1 用于在obs中pdf和笔记
屏幕2 用于 写作
因为知识卡全部在一个库里。如何能打开一个库 在两个屏幕中
eg：目标库：A  我需要在屏幕1中打开库A 同时在屏幕2中也打开库A

**Comment by Tony-zf1** [comment]
百度“文件夹映射”，再打开映射的库，相当于同一个库双开了
